testscript/testdata/testscript_update_script_not_in_archive.txt

```diff
@@ -0,0 +1,12 @@
+unquote scripts/testscript.txt
+cp scripts/testscript.txt unchanged
+testscript-update scripts
```

Comment:
Shouldn't this line need to be negated? Because the file does not exist in the archive, hence it should fall through to comparing and that should fail?

Reply:
Excellent catch, sir! There was a bug in the fakeT logic (it never set the failed field). Fixed now.